pkgs/tools/text/ansifilter/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, pkgs }:
```

Comment:
Please pass in specific members of `pkgs`, passing in the whole package set is bad practice.